[2024-12-26T18:30:57.992+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-12-26T18:30:58.016+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bash_dag.bash_ex2 scheduled__2024-12-25T08:00:00+00:00 [queued]>
[2024-12-26T18:30:58.024+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bash_dag.bash_ex2 scheduled__2024-12-25T08:00:00+00:00 [queued]>
[2024-12-26T18:30:58.024+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-12-26T18:30:58.040+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): bash_ex2> on 2024-12-25 08:00:00+00:00
[2024-12-26T18:30:58.052+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=69) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-26T18:30:58.053+0000] {standard_task_runner.py:63} INFO - Started process 73 to run task
[2024-12-26T18:30:58.054+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'bash_dag', 'bash_ex2', 'scheduled__2024-12-25T08:00:00+00:00', '--job-id', '1084', '--raw', '--subdir', 'DAGS_FOLDER/dag1.py', '--cfg-path', '/tmp/tmpdwxkv0z1']
[2024-12-26T18:30:58.056+0000] {standard_task_runner.py:91} INFO - Job 1084: Subtask bash_ex2
[2024-12-26T18:30:58.098+0000] {task_command.py:426} INFO - Running <TaskInstance: bash_dag.bash_ex2 scheduled__2024-12-25T08:00:00+00:00 [running]> on host 87a1e11545b0
[2024-12-26T18:30:58.190+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='bash_dag' AIRFLOW_CTX_TASK_ID='bash_ex2' AIRFLOW_CTX_EXECUTION_DATE='2024-12-25T08:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-25T08:00:00+00:00'
[2024-12-26T18:30:58.191+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-12-26T18:30:58.203+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-12-26T18:30:58.204+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Task 2 now $(date)"']
[2024-12-26T18:30:58.214+0000] {subprocess.py:86} INFO - Output:
[2024-12-26T18:30:58.217+0000] {subprocess.py:93} INFO - Task 2 now Thu Dec 26 18:30:58 UTC 2024
[2024-12-26T18:30:58.217+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-12-26T18:30:58.218+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-12-26T18:30:58.241+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=bash_dag, task_id=bash_ex2, run_id=scheduled__2024-12-25T08:00:00+00:00, execution_date=20241225T080000, start_date=20241226T183058, end_date=20241226T183058
[2024-12-26T18:30:58.269+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-12-26T18:30:58.279+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-26T18:30:58.281+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
